java/java.editor/src/org/netbeans/modules/editor/java/JavaCompletionCollector.java

```diff
@@ -651,7 +652,7 @@ public Completion createAttributeValueItem(CompilationInfo info, String value, S
                 }
             }
             Builder builder = CompletionCollector.newBuilder(label)
-                    .kind(Completion.Kind.Text)
+                    .kind(Completion.Kind.Value)
                     .sortText(value)
```

Comment:
something in this PR, possibly this snippet is causing a regression: https://github.com/apache/netbeans/issues/7522

Reply:
sorry for the noise, I can't reproduce it with the nightly build. This means my dev build didn't clean properly while switching branches or something similar - my mistake.